manifests/compute_node/config.pp

```diff
@@ -69,13 +69,15 @@
     source => 'puppet:///modules/one/sudoers_imaginator',
   }
 
-  file { '/sbin/brctl':
-    ensure => link,
-    target => '/usr/sbin/brctl',
+  if $::osfamily == 'Debian' {
```

Comment:
I can't really argue with that :) It might finally be time that I learned how to write rspec tests...